doc/connectivity/functional_connectomes.rst

```diff
@@ -105,38 +105,40 @@ regions, regressing out noise sources is indeed very important
 
    Try using the information above to compute the correlation matrix of
    the first subject of the ADHD dataset downloaded with
-   :func:`nilearn.datasets.fetch_adhd`
+   :func:`nilearn.datasets.fetch_adhd`.
 
    **Hints:**
 
    * Inspect the '.keys()' of the object returned by
-     :func:`nilearn.datasets.fetch_adhd`
+     :func:`nilearn.datasets.fetch_adhd`.
 
    * :func:`numpy.corrcoef` can be used to compute a correlation matrix
-     (check the shape of your matrices)
+     (check the shape of your matrices).
 
-   * :func:`matplotlib.pyplot.imshow` can show a correlation matrix
+   * :func:`matplotlib.pyplot.imshow` can show a correlation matrix.
 
-   * The example above has the solution
+   * The example above has the solution.
 
 |
 
 Time-series from a probabilistic atlas
-========================================
+======================================
 
 Probabilistic atlases
-----------------------
+---------------------
 
 The definition of regions as by a continuous probability map captures
 better our imperfect knowledge of boundaries in brain images (notably
 because of inter-subject registration errors). One example of such an
 atlas well suited to resting-state data analysis is the `MSDL atlas
-<https://team.inria.fr/parietal/research/spatial_patterns/spatial-patterns-in-resting-state/>`_ (:func:`nilearn.datasets.fetch_atlas_msdl`).
+<https://team.inria.fr/parietal/research/spatial_patterns/spatial-patterns-in-resting-state/>`_
+(:func:`nilearn.datasets.fetch_atlas_msdl`).
 
 Probabilistic atlases are represented as a set of continuous maps, in a
 4D nifti image. Visualization the atlas thus requires to visualize each
 of these maps, which requires accessing them with
-:func:`nilearn.image.index_img` (see the :ref:`corresponding example <sphx_glr_auto_examples_01_plotting_plot_overlay.py>`).
+:func:`nilearn.image.index_img` (see the :ref:`corresponding example
+<sphx_glr_auto_examples_01_plotting_plot_overlay.py>`).
```

Comment:
No, sphinx, as latex, usually don't care for line splitting (except if it between markup). In this case, for links, it's ok to split the line before the `<...>_` part.